### PR TITLE
Feature preview builds

### PR DESCRIPTION
**Pull request by MCMrARM-Win**

### CODE REVIEW

**Comment by MCMrARM** [comment]
Note: preview builds unlike beta don't require Insider signup or an account to work, and therefore the token code is not applicable there.

File: MCLauncher/VersionList.cs [hunk 1]
```diff
@@ -49,7 +49,10 @@ public class VersionList : ObservableCollection<WPFDataTypes.Version> {
             // ([name, uuid, versionType])[]
             foreach (JArray o in data.AsEnumerable().Reverse()) {
                 bool isNew = dbVersions.Add(o[0].Value<string>()) && !isCache;
-                Add(new WPFDataTypes.Version(o[1].Value<string>(), o[0].Value<string>(), (WPFDataTypes.VersionType) o[2].Value<int>(), isNew, _commands));
+                int versionType = o[2].Value<int>();
+                if (!Enum.IsDefined(typeof(WPFDataTypes.VersionType), versionType) || versionType == (int) WPFDataTypes.VersionType.Imported)
```
Comment (dktapps) [inline MCLauncher/VersionList.cs:53]: isn't `Imported` already a member of `VersionType`?

Comment (MCMrARM) [reply MCLauncher/VersionList.cs:53]: What do you mean by this? I added Imported to VersionType, and this line forbids the server from using the Imported VersionType.

Comment (dktapps) [reply MCLauncher/VersionList.cs:53]: My bad, I misunderstood the code.

**Pull request merged**